docs/.pages

```diff
@@ -5,6 +5,9 @@ nav:
     - Smash Remix: remix.md
     - Server Status: server-status.md
     - P2P Checker: p2p-check.md
+    - Controller Setup Guides:
```

Comment:
The order of this should probably match the order of the main page. So putting GameCube then Xbox would be fine.

Reply:
Fixed in https://github.com/smash64-dev/smash64-online/pull/10/commits/a7ff7fd5e34b84199d0cc4fd3adcfa3422f6df3f